oak_functions_service/src/lookup_htbl.rs

```diff
@@ -0,0 +1,575 @@
+//
+// Copyright 2024 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+// This is a key/value hash table optimized for large numbers of small k/v pairs.  We assume we
```

Comment:
Formatting nits, but a lot of these comments should be written/formatted in `rustdoc` style.

Reply:
Agreed... which means I should go read about rustdoc :)

Reply:
Done.